src/core/model/schema.ts

```diff
@@ -135,6 +135,19 @@ export const toPrintableObject = (schemaElement: SchemaElement): any => ({
   ]),
 });
 
+export const toPrintableDebugObject = (debugSchema: SchemaElement): any => {
+  return assign(debugSchema, {
+    parent: debugSchema.parent?.uuid,
+    children: Array.from(containsAs(debugSchema)).map(([el, key]) => [
+      key,
+      toPrintableDebugObject(el),
+    ]),
```

Comment:
Instead of this virtual 'children', shouldn't we print it exactly as it is in the element?

So pretty-print the content of the `properties` map, `items`(object or array) and `other` map. I don't think we have other cases than them. We could still print the `children` (but only their names and not their content) if you'd like to, but I would use the `getChildren` method for that as it's also used in the tree for example.

Reply:
Fixed it.